apps/block_scout_web/lib/block_scout_web/controllers/html_controller.ex

```diff
@@ -1,14 +1,17 @@
 defmodule BlockScoutWeb.HtmlController do
     use BlockScoutWeb, :controller
     alias Phoenix.View
-    alias Explorer.Stats
-
-    def index(conn, _params) do
-        render(
-            conn,
-            "index.html",
-            current_path: current_path(conn),
-        )
+    alias BlockScoutWeb.Call
+    alias BlockScoutWeb.Controllers
+    
+
+
+    def show(conn, %{"id" => id}) do
```

Comment:
is this necessary?

Reply:
html_controller is only for testing purposes. I will remove this when everything is done, the functionality is added in the transaction_internal_transaction_controller.ex